public/data/umaresso.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Umar Khatab",
+  "type": "personal",
+  "bio": "React Js developer",
```

Comment:
```suggestion
  "bio": "React JS developer",
```

Reply:
@umaresso You can also replace it with Javascript, but for now its ok as it is, just capitalize the text.

Reply:
okey , i have done the suggested changes. Now i'm going to close the issue and start a new One